src/types.rs

```diff
@@ -27,8 +27,8 @@ pub enum EdgeKind {
     ControlFlow,
     /// Data edges of a DDG region, also known as "wires".
     Value(SimpleType),
-    /// A reference to a constant value definition, used in the module region.
-    Const(ClassicType),
+    /// A reference to a static value definition, used in the module region.
```

Comment:
Drive-by again, but I think now that we have local constants you could remove ", used in the module region" ?